compiler/annotation/type_seeder.rs

```diff
@@ -229,6 +230,7 @@ impl<'this, Snapshot: ReadableSnapshot> TypeGraphSeedingContext<'this, Snapshot>
                 Constraint::FunctionCallBinding(c) => c.apply(self, vertices)?,
                 Constraint::RoleName(c) => c.apply(self, vertices)?,
                 Constraint::Value(c) => c.apply(self, vertices)?,
+                | Constraint::Iid(_)
```

Comment:
IIDs are extracted into parameters, so we can't get any type information from them.